independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json

```diff
@@ -0,0 +1,514 @@
+          }
+        }
+      }
+    },
+    "/zoek/lijnrichtingen/{zoekArgument}": {
+      "get": {
+        "summary": "zoekLijnrichtingen",
+        "description": "Search lines by number or description. Eg search line 48 or Hamont",
+        "operationId": "ZoekLijnrichtingen",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "zoekArgument",
+            "description": "het zoekargument",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "huidigePositie",
+            "description": "de latitude en longitude coordinaat waarden gescheiden door komma bv, 51.056064,3.797336 van de lokatie waarvoor lijnen in de buurt gezocht wordt",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "startIndex",
+            "description": "Format - int32. de start positie in de lijst van gevonden lijnen",
+            "type": "integer",
+            "default": 0
+          },
+          {
+            "in": "query",
+            "name": "maxAantalHits",
+            "description": "Format - int32. het maximaal aantal lijnen aangeleverd door deze service ",
+            "type": "integer",
+            "default": 10
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/LijnRichtingHits"
+            },
+            "examples": {
+              "application/json": {
+                "aantalHits": 6,
+                "lijnrichtingen": [
+                  {
+                    "lijnNummerPubliek": "335",
+                    "entiteitnummer": "3",
+                    "lijnnummer": "335",
+                    "richting": "HEEN",
+                    "omschrijving": "Leuven - Aarschot",
+                    "bestemming": "LEUVEN GASTHUISB.",
+                    "kleurVoorGrond": "#FFFFFF",
+                    "kleurAchterGrond": "#991199",
+                    "kleurAchterGrondRand": "#991199",
+                    "links": [
+                      {
+                        "rel": "string",
+                        "url": "string"
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          },
+          "500": {
+            "description": "service of achterliggende systemen niet beschikbaar"
+          }
+        }
+      }
+    },
+    "/zoek/locaties/{zoekArgument}": {
+      "get": {
+        "summary": "zoekLocaties",
+        "description": "Search locations (x,y) by user adress input. Typical for input on a routeplanner (eg kerkstraat 14 Hasselt or Achter De Kazerne ) . Limitied Geocoding service that retuns stops, adresses or POIs and its XY coordinates",
```

Comment:
@Lschockaert all files should be free of grammatical errors and typos. Please change the following
"adress" to "address"
"Limitied" to "Limited"
"retuns" to "returns"
"adresses" to "addresses"